packages/std/src/lib.rs

```diff
@@ -5,6 +5,26 @@ Builds without the std feature are currently not expected to work. If you need n
 "#
 );
 
+/// Implements a hidden constructor for query responses.
+// #[macro_export]
+macro_rules! impl_hidden_constructor {
+    ( $response:ty, $( $field: ident : $t: ty),* ) => {
+        impl $response {
+            /// Constructor for testing frameworks such as cw-multi-test.
+            /// This is required because query response types should be #[non_exhaustive].
```

Comment:
Now that this is not only used for query responses, the doc comment needs some adjustment.
```suggestion
            /// This is required because this type should be #[non_exhaustive].
```